src/stdatm/atmosphere.py

```diff
@@ -15,7 +15,7 @@
 #  along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 from copy import deepcopy
-from numbers import Number
+from numbers import Number, Real
```

Comment:
Is Number still necessary ? I thought it was too generic

Reply:
Right. Got rid of these remnants in 021bb5cdf51def27701eb734f4ecef1278f8a74b